From dc2796d724e6bf439fdcfbc7552ca16af10bc56e Mon Sep 17 00:00:00 2001
From: Bartosz Golaszewski <brgl@bgdev.pl>
Date: Thu, 10 Feb 2022 10:28:44 +0100
Subject: [PATCH 1/2] build: don't use -Werror

Using -Werror is the best way to waste a lot of time and not get stuff
done. Drop it.

Signed-off-by: Bartosz Golaszewski <brgl@bgdev.pl>
---
Upstream-Status: non-applicable

 cmake/compiler_settings.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/compiler_settings.cmake b/cmake/compiler_settings.cmake
index 1fc3b073ed..6fcf515b0c 100644
--- a/cmake/compiler_settings.cmake
+++ b/cmake/compiler_settings.cmake
@@ -53,7 +53,7 @@ macro(set_gcc_flags)
 endmacro()
 
 macro(set_gcc_warnings)
-    list(APPEND AWS_COMPILER_WARNINGS "-Wall" "-Werror" "-pedantic" "-Wextra" "-Wshadow")
+    list(APPEND AWS_COMPILER_WARNINGS "-Wall" "-pedantic" "-Wextra" "-Wshadow")
     if(COMPILER_CLANG)
         if(PLATFORM_ANDROID)
             # when using clang with libc and API lower than 21 we need to include Android support headers and ignore the gnu-include-next warning.
-- 
2.30.1

